lib/components/Carleton360/Examples.stories.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react'
+import type { Meta, StoryObj } from '@storybook/react'
+import { Section } from '../../layouts/Section/Section'
+import { Main } from '../../layouts/Main/Main'
+import { FooterStandard } from '../Footer/FooterStandard/FooterStandard'
+import { Column } from '../../layouts/Column/Column'
+import { Nav } from '../Nav/Nav'
+import { ButtonGroup } from '../ButtonGroup/ButtonGroup'
+import { Button } from '../Button/Button'
+import { NavButtonsData } from '../../data/NavData'
+import { Carleton360 } from './Carleton360'
+
+const meta: Meta = {
+  title: 'Components/Carleton360/Examples',
+  parameters: {
+    layout: 'fullscreen',
+  },
+}
+
+export default meta
+type Story = StoryObj
+
+const SinglePara = () => {
+  return (
+    <>
+      <p>
+        Nobis voluptatem dolorum et eum doloremque cupiditate velit. Praesentium architecto a distinctio aut
+        reprehenderit ducimus. Perferendis excepturi delectus nihil voluptatem non. Molestiae quas dolores accusamus in.
+        Praesent quis ligula quis nulla malesuada tempor.
+      </p>
+    </>
+  )
+}
+
+export const LayoutExamples: Story = {
+  render: () => (
+    <>
+      <Nav>
+        <Nav.Top>
+          <Nav.Logo title="Raven Design System" link="/" />
+          {/* <Nav.Menu menu={NavDataSingle} /> */}
+          <Nav.Aside>
+            <Nav.Buttons menu={NavButtonsData} />
+          </Nav.Aside>
+        </Nav.Top>
+      </Nav>
+
+      <Main maxWidth="5xl">
+        <Section maxWidth="max">
+          <Carleton360 headerType="h1">
+            <ButtonGroup>
+              <Button title="Apply Now" />
+              <Button color="dark-grey" title="Request Information" />
+            </ButtonGroup>
+          </Carleton360>
+        </Section>
```

Comment:
This should move down the page so we don't misrepresent this as an element that will be available to use as a page banner. We can just add a default TextImage with just the title and place the 360 after a single para